src/main/scala/mesosphere/marathon/util/heartbeat/HeartbeatActor.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.util.heartbeat
```

Comment:
Feels like `MesosHeartbeatMonitor` and `HeartbeatActor` have a close relationship and should live in the same package. We try to make components under `core` perhaps `core/heartbeat`?